deno/payloads/v10/channel.ts

```diff
@@ -426,6 +426,7 @@ export enum ChannelType {
 	 *
 	 * @deprecated This is the old name for {@apilink ChannelType#AnnouncementThread}
 	 */
+	// eslint-disable-next-line @typescript-eslint/no-duplicate-enum-values
```

Comment:
Should we disable this rule globally?

Reply:
It's only used in 2 places (excluding /deno). I think it may be fine for now.